client/src/components/DownloadOptions.js

```diff
@@ -5,12 +5,16 @@ import { sortArrayByKey } from 'helpers/sortArrayByKey'
 
 export const DownloadOptions = ({ resource, handleSelectFile }) => {
   const { computed_files: computedFiles } = resource
+
+  // TODO: Remove filter when ANN_DATA lands
```

Comment:
I won't comment on the code blocks marked as `// TODO: Remove filter when ANN_DATA lands`, since they'll be cleaned up in later AnnData PRs (e.g., the no longer used `type` field  value `PROJECT_MULTIPLEXED_ZIP`).

Reply:
Yeah this will be removed, temporarily added so that staging can be deployed to staging where AnnData exists on the backend but not the client. This will be removed when the AnnData content on the backend is supported on the client. Basically, the existing client doesn't know how to present AnnData computed files, so filtering them out restores the client side functionality.